index.js

```diff
@@ -60,7 +66,12 @@ function spawnClangFormat(args, done, stdio) {
     // remove glob from arg list
     args = args.filter(function(arg){return arg.indexOf('--glob=') === -1;});
 
-    return glob(filesGlob, function(er, files) {
+    glob(filesGlob, function(err, files) {
```

Comment:
the code above in `clangFormat` calls this method and then returns `.stdout` from the process. That will fail with this change, but admittedly wasn't working properly before (it'd return `undefined` if filesGlob was set, now it'll crash).

Can you take a look at that? I think it's OK to return undefined if filesglob is set, it doesn't make sense to return "the formatted code" if we're formatting files. But we should not crash.

Reply:
It would have crashed either way, unless `glob` returns something (which I believe it doesn't). That being said, this line is unreachable from `clangFormat` because it doesn't add a `--glob` parameter, so `filesGlob` will never have a truthy value.

I would be in favor of having `spawnClangFormat` consistently return a value (a `Readable` seems to make the most sense). But that would be a breaking change... what do you think about this?

Reply:
BTW, I would be happy to help with any potential API changes/corrections.

Reply:
You're right, that's a "cannot happen" condition. Maybe rather `throw new Error()` than returning undefined? That way we keep a more reasonable API and catch the error issue.

Reply:
Done. I also added the call to `setEncoding` since `enc` is otherwise unused